hw/xbox/nv2a/nv2a_regs.h

```diff
@@ -256,6 +256,10 @@
 #   define NV_PGRAPH_INCREMENT_READ_3D                          (1 << 1)
 #define NV_PGRAPH_FIFO                                   0x00000720
 #   define NV_PGRAPH_FIFO_ACCESS                                (1 << 0)
+#define NV_PGRAPH_RDI_INDEX                              0x00000750
+#   define NV_PGRAPH_RDI_INDEX_ADDRESS                        0x00000FFC
```

Comment:
Are we sure this isn't `0x1FFC`? MMIO region is 0x2000, so it looks suspicious

Reply:
The mask is what D3D debug libs use.

What MMIO region? RDI probably exists because there's no MMIO to this GPU RAM.

This is an index into the selected bank (`NV_PGRAPH_RDI_INDEX_SELECT` to pick bank of a feature). So there can be up-to 1024 words per bank, but probably not all of those are used for each feature.

If you pick the bank for the constant space bank for example, it probably only has 192 vec4 constants = 768 valid indices.
The program space bank should only have 136 * 4 words per instruction = 544 valid indices.
\- So the exact size of each bank could be different.

I'm not sure what happens at higher indices, but we shouldn't have to worry about it yet; I have added an `assert` to guard against higher indices in the constant banks.